isis/src/base/apps/topds4/topds4.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="topds4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
+  <brief>
+    Convert from ISIS3 cube to PDS 4 format using template XML files
+  </brief>
+
+  <description>
+    <p>
+      Writes a PDS4 compatible label file and image file. The contents of the label file are
+      generated useing the template file specified by the TEMPLATE parameter.
+    </p>
+    <p>
+      The image file is generated from the input ISIS cube specified by the FROM parameter.
+      The output image file will a copy of the input ISIS cube with the image data stored in
+      Band Sequential format.
+    </p>
+    <p>
+      This application uses the <a href="https://pantor.github.io/inja">Inja templating engine</a>
+      to render the template file. The input data from the cube and extra data sources
+      is converted to JSON and then it is accessed via the Inja templating syntax.
+      See the <a href="https://pantor.github.io/inja">Inja documentation</a> for the full template
+      syntax. The data from the label of the input cube is accessible under "MainLabel",
+      the data from the original PVL or XML based label of the input cube is accessible 

```

Comment:
I'd put something here about how this comes from the originally ingested label.

Reply:
Oops, the previous comment should be in response to this post.

Reply:
Bah, never mind. I just clicked on the link in the text and it did bad things. I think it is OK. Sorry for the false alarm.